src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -160,7 +155,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <!--Define platform WinMD references for modern IDL compilation-->
     <Target Name="GetCppWinRTPlatformWinMDReferences"
-            DependsOnTargets="ResolveAssemblyReferences"
+            DependsOnTargets="ResolveAssemblyReferences;$(GetCppWinRTPlatformWinMDReferencesDependsOn)"
```

Comment:
another orphan DependsOn prop

Reply:
I would say some targets are us hooking into MSBuild to make functionality work, like fixing the references for static libs. I don't think those targets should have DependsOn.